images/linux/toolsets/toolset-1804.json

```diff
@@ -67,6 +67,7 @@
             "platform" : "linux",
             "arch": "x64",
             "versions": [
+                "*",
                 "*"
```

Comment:
What's the implication of having two `*`s here?  Also, do we want to update the macOS toolsets too?

Reply:
Hm, I was hoping that the two `*`s would test that there are two versions (each of which can be any version). I'm not sure if just one `*` would have the same meaning.

It looks like CodeQL doesn't exist in the macOS toolsets right now, which is why I haven't added them. I think for macOS we are relying on the `Common.Tests` file (and invoking the test at the end of the script for the macOS file directly).

Reply:
Resolved below 👍